app/src/androidTest/java/tech/pauly/findapet/MainTabActivityTest.java

```diff
@@ -0,0 +1,57 @@
+package tech.pauly.findapet;
+
+import android.support.test.rule.ActivityTestRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+public class MainTabActivityTest {
+
+    @Rule
+    public ActivityTestRule<MainTabActivity> activityRule = new ActivityTestRule<>(MainTabActivity.class);
+
+    @Test
+    public void onLaunch_showDiscover() {
+        RobotUtils.seesView(R.id.fragment_discover);
+    }
+
+    @Test
+    public void onClickDiscover_seeOnlyDiscover() {
+        RobotUtils.clickView(R.id.navigation_discover);
+
+        RobotUtils.seesView(R.id.fragment_discover);
```

Comment:
Definitely want to move to a Robot pattern to clean these up and be more expressive.

Reply:
To be picked up in https://www.pivotaltracker.com/story/show/157172865